### PR TITLE
Add first round of grants blog post

### PR DESCRIPTION
**Pull request by bitschmidty**
- adds grantees to programs page
- adds blog post announcing grantees

### CODE REVIEW

File: _posts/2021-03-31-grant-announcements.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<br style="clear:both" />
+
+## Hennadii Stepanov
+
+<img src="/assets/images/hennadii-stepanov.jpg" alt="Picture of Hennadii Stepanov" style="float:left;border:1px solid
+darkgray;margin-right:25px;width:150px;" />
+Hennadii has been contributing code and review to Bitcoin Core since 2018. In that time he
+has become [one of the most prolific contributors to the project][hebasto]. Hennadii’s funding allows
+him to work full time on Bitcoin Core development and review. Hennadii has previously
+received support from [CoinCards][coincards] and [Payvant][payvant].
```
Comment (glozow) [inline _posts/2021-03-31-grant-announcements.md:51]: I don't know any context, but should this part be in italics like the Nexo comment for Jesse?

**Review by glozow — COMMENTED** [review]
did a `make preview`, looks nice

File: _posts/2021-03-31-grant-announcements.md (hunk 1 above)
Comment (bitschmidty) [reply _posts/2021-03-31-grant-announcements.md:51]: @glozow Nexo is sponsoring this go around but CoinCards and Payvant were historical sponsors for hebasto.

File: _data/team.yml [hunk 2]
```diff
@@ -105,3 +105,36 @@ fellows:
...
+    bring additional security, privacy and functionality to multisignature and
+    second layer protocols.
+  social: "@jesseposner"
+  link: https://twitter.com/jesseposner
+  image: assets/images/jesse-posner.jpg
+  alt: Picture of Jesse Posner
+
+- name: Alekos Filini
+  description: >-
+    Alekos’s funding supports his work as maintainer of the Bitcoin Developer
```
Comment (harding) [inline _data/team.yml:123]: Is "Developer" actually in the name?  Looking at the website and the README for the repository, I only see "Dev".  The LN kit is "Lightning *Development* Kit", so I'd have expected this to use basically the same name.

If you change this, it also needs to be updated in the blog post.

File: _posts/2021-03-31-grant-announcements.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+_Grant made possible by very generous financial support from [Nexo][nexo]._
+<br style="clear:both" />
+
+## Alekos Filini
+
+<img src="/assets/images/alekos-filini.jpg" alt="Picture of Alekos Filini" style="float:left;border:1px solid
+darkgray;margin-right:25px;width:150px;" />
+Alekos's funding supports his work as maintainer of the [Bitcoin Developer Kit][bdk]. BDK is a
+lightweight and modular collection of tools for use by wallets and other Bitcoin applications.
```
Comment (harding) [inline _posts/2021-03-31-grant-announcements.md:38]: Nit: "collection of tools" sounds to me like a collection of binaries.  Maybe call it a "library" or a "toolkit"?

**Review by harding — COMMENTED** [review]
Reviewed code and quickly reviewed previews of the blog post and updated Programs page; LGTM with one concern, one nitpick.  Didn't check any links.

File: _posts/2021-03-31-grant-announcements.md (hunk 1 above)
Comment (jnewbery) [reply _posts/2021-03-31-grant-announcements.md:51]: That's right. I wanted to give CoinCards and Payvant some recognition for their generous support of Hennadii over the last year and a bit. They're both small companies, but have given more than many much larger companies in the space.

File: _data/team.yml (hunk 2 above)
Comment (jnewbery) [reply _data/team.yml:123]: :facepalm: my mistake. I've changed it to "Bitcoin Dev Kit" in both places.

File: _posts/2021-03-31-grant-announcements.md (hunk 3 above)
Comment (jnewbery) [reply _posts/2021-03-31-grant-announcements.md:38]: This was taken (slightly modified) from https://bitcoindevkit.org/:

> The Bitcoin Dev Kit (BDK) project [...] aims to build a collection of tools and libraries [..] All BDK components are designed to be lightweight and modular.

**Comment by jnewbery** [comment]
This looks good to me. I've made some minor edits, and added a sentence trailing the next three blog posts.

**Comment by bitschmidty** [comment]
ACK on minor updates commit

**Pull request merged**